### DISCUSSION TITLE
What to learn after Solidity?

### DISCUSSION THREAD

**Discussion by MuhammadAmmar24** [nehal2210/Blockchain-Engineer-Course#13] [Q&A] [answered]
After completing solidity course, what a person should learn if he doesn't know javascript?

**Comment by TacitvsXI** [comment, marked as answer]
After completing a Solidity course, if a person does not know JavaScript, they should consider learning it. JavaScript is an essential programming language for developing decentralized applications on the Ethereum blockchain. However, the specific goals of the developer will also play a significant role in determining what they should learn next. For example, understanding core Blockchain concepts is crucial for those interested in developing advanced blockchain applications rather than just NFTs and Coins. In any case, starting with JavaScript as a second language after Solidity is a good direction to pursue :)

**Reply by Muhammad-Ali-bhatti** [reply]
@nehal2210 @ILESKOV Bhai I have A Confusion Regarding My career. I Am 20 year old and student of 2nd year. I have a question What should I do after learning Solidity how I can get internship or job for practical experience So I Can Boost My career In Blockchain Development. The problem is that if I don't have any solid grounds too show my family Something they will Force Me To do  something else and I will stuck in between different things. Can you Advice Me Something to Your Younger Brother

**Reply by TacitvsXI** [reply]
Hey, @Muhammad-Ammar-Masood  Brother. I understand that it can be tough to navigate career decisions, especially when it comes to getting support from family. It's important to communicate with them and let them know that you're exploring different options to find what's right for you. You can explain to them that blockchain development is a growing field with many exciting opportunities, and that your skills in Solidity and other programming languages can be valuable in this industry:)

In terms of exploring blockchain development, there are many areas to consider, such as blockchain with front-end development, blockchain with back-end development, or even auditing.  I recommend taking some time to explore these areas and see which ones align with your interests and strengths. 

Remember, it's okay to take some time to figure out what you want to do. Don't be afraid to take risks and pursue your passion, even if it doesn't align with your family's expectations at first. With hard work and determination, you can build a successful career in blockchain development. If you'd like more guidance on your career goals, I'm happy to have a personal chat with you. Feel free to email me at [ivan.leskov@protonmail.com](mailto:ivan.leskov@protonmail.com).

**Reply by nehal2210** [reply]
I can understand your situation, the first thing is that you should put more focus on your 2nd-year exam and take admission on a reputable University.
Technology is continuously changing, So you should need to focus on becoming a good software engineer and be flexible enough to adopt changes. (means  make your general coding  and computer science concepts stronger)

For Blockchain, perspective starts taking part on Blockchain hackathons even if you do not know anything just take part and meet people
join communities on Discord eventually, you will get the direction.

For Quick Internship and job start learning web development in React or any framework and starts applying on companies,  give an interview to them and fail eventually you know what are the questions they ask, and then cracking the interview is become a piece of cake. for you.

###########################################
SAFAR BHT LMBA HE
#########################################

for the satisfaction of your parents, you should do some part-time job like giving tuition until you get the job. sb ko krna prta he bhai preshan na ho bhai.

Summary
1 saal lgaao ache se web-development and Blockchian integration me job mil jaae gi bs apply krte rehna interview dete rehna.

**Reply by Muhammad-Ali-bhatti** [reply]
> I can understand your situation, the first thing is that you should put more focus on your 2nd-year exam and take admission on a reputable University.
> 
> Technology is continuously changing, So you should need to focus on becoming a good software engineer and be flexible enough to adopt changes. (means  make your general coding  and computer science concepts stronger)
> 
> 
> 
> For Blockchain, perspective starts taking part on Blockchain hackathons even if you do not know anything just take part and meet people
> 
> join communities on Discord eventually, you will get the direction.
> 
> 
> 
> For Quick Internship and job start learning web development in React or any framework and starts applying on companies,  give an interview to them and fail eventually you know what are the questions they ask, and then cracking the interview is become a piece of cake. for you.
> 
> 
> 
> ###########################################
> 
> SAFAR BHT LMBA HE
> 
> #########################################
> 
> 
> 
> for the satisfaction of your parents, you should do some part-time job like giving tuition until you get the job. sb ko krna prta he bhai preshan na ho bhai.
> 
> 
> 
> Summary
> 
> 1 saal lgaao ache se web-development and Blockchian integration me job mil jaae gi bs apply krte rehna interview dete rehna.
> 
> 

Thank You Bhai Next month paper ha Ap dua Karna Inshallah I will start small And make it Big Thank You So much
